packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -491,19 +481,17 @@ function _finalizeElementPosition(
   bounds?: Rectangle,
   alignmentEdge?: RectangleEdge,
   coverTarget?: boolean,
-  previousPositions?: boolean
+  doNotFinalizeReturnEdge?: boolean
```

Comment:
It's my own bias, but I think functions are easier to understand if they don't take boolean flags as args. They tend to accumulate over time and are not easy to grok when looking at invocations.

Since this method is small, can there be 2 methods instead of an `if`? I think this lets you walk back several changes and have `_positionCard` make a more straightforward method call all the way through.

Reply:
I think that we could do that in the future. For now it's hidden enough that it should be okay.